bundles/org.openhab.binding.tibber/README.md

```diff
@@ -209,13 +214,13 @@ Example of tomorrow and today prices data structure - an array of tuples:
 
 ### demo.things
 
-```java
+```
```

Comment:
Please revert this change for proper syntax highlighting.

Reply:
done